### PR TITLE
frontend maps

### PR DESCRIPTION
**Pull request by toep**
* Add dialog when a driver arrives at users location.
* Add dialog when driver/user arrives at destination - tells user how much money is charged.
* Changed car icon to SVG that is rotated based on direction moving.
* Increased animation time from every 250 ms to every 50 ms for a smoother ride.
* Display a blue ball of users current location.
* Fixed axios requests to match Suraj's changes.
* Send axios request ARRIVAL when arrived at destination.
* Redirect user to login page if no session is found

### CODE REVIEW

File: packages/frontend/src/components/Dashboard/AccountInfo/index.jsx [hunk 1]
```diff
@@ -33,9 +31,8 @@ export default class AccountInfoView extends Component {
     this.setState({
       name: `${response.data.firstName} ${response.data.lastName}`,
```
Comment (vikr01) [inline packages/frontend/src/components/Dashboard/AccountInfo/index.jsx]: Note that for values like `undefined` or `null`, forcing them into a string will form `"undefined"` or `"null"`.

File: packages/frontend/src/components/Dashboard/Overview/AlertDialog.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+              Ok
+            </Button>
+          </DialogActions>
+        </Dialog>
+      </div>
+    );
+  }
+}
+
+AlertDialog.defaultProps = {
```
Comment (vikr01) [inline packages/frontend/src/components/Dashboard/Overview/AlertDialog.jsx]: This needs to be changed to `static defaultProps = {...}` in the class.

File: packages/frontend/src/components/Dashboard/AccountInfo/index.jsx (hunk 1 above)
Comment (toep) [reply packages/frontend/src/components/Dashboard/AccountInfo/index.jsx]: Yes I noticed that. However, this won't be an issue as we check if we're logged in before this page gets to be shown. So the backend should always return valid data here.

File: packages/frontend/src/components/Dashboard/Overview/CostEstimater.jsx [hunk 3]
```diff
@@ -11,16 +11,20 @@ type Props = {
...
   <Typography variant="h5">
     Cost:
     {' $'}
-    {(
-      START_COST +
-      (COST_PER_MILE *
-        (meters - FIRST_N_MILES_FREE < 0 ? 0 : meters - FIRST_N_MILES_FREE)) /
-        METERS_TO_MILES
-    ).toFixed(2)}
+    {estimateCost(meters).toFixed(2)}
```
Comment (vikr01) [inline packages/frontend/src/components/Dashboard/Overview/CostEstimater.jsx]: I've had issues with `toFixed` before, we'll keep it like this for now if it's working.

File: packages/frontend/src/components/Dashboard/Overview/GMapsControl.jsx [hunk 4]
```diff
@@ -22,8 +25,131 @@ const GMapsControl = compose(
   withScriptjs,
   withGoogleMap,
   lifecycle({
+    componentDidUpdate() {
```
Comment (vikr01) [inline packages/frontend/src/components/Dashboard/Overview/GMapsControl.jsx]: This is really confusing. We should see if we can just use a regular React component here later.

Comment (toep) [reply packages/frontend/src/components/Dashboard/Overview/GMapsControl.jsx]: Yeah, I know... It was the only way the react-google-maps had examples, and was unable to get it to work with a class.

**Pull request merged**